server/src/main/kotlin/pishello/hello/api/AuthEndpoint.kt

```diff
@@ -8,6 +8,7 @@ import pishello.hello.persistence.database.ports.TokenPort
 import pishello.hello.persistence.database.ports.UserPort
 
 data class LoginRequest(val name: String, val password: String)
+data class RegisterRequest(val name: String, val password: String, val passwordRepeat: String)
```

Comment:
skoro sprawdzamy na frontendzie, to ta informacja o passwordRepeat nie jest już nam w serwerze potrzeba, tym bardziej jak jej nie wykorzystujesz, prośba o usunięcie ;)

Reply:
Usunięte